<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileCollectDialog</class>
 <widget class="QDialog" name="FileCollectDialog">
  <property name="windowTitle">
   <string>File transfer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="filetransfer.qrc">
    <normaloff>:/filetransfer/collect-files.png</normaloff>:/filetransfer/collect-files.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Collected files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressBar"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Overall progress</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTreeView" name="collectionsTreeView">
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>300</number>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>300</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputDirectoryGroupBox">
     <property name="title">
      <string>Output directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="outputDirectoryEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="openOutputDirectoryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="icon">
         <iconset resource="../../core/resources/core.qrc">
          <normaloff>:/core/document-open.png</normaloff>:/core/document-open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../core/resources/core.qrc"/>
  <include location="filetransfer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileCollectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileCollectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
